### PR TITLE
:memo::art: lint Mutations

### PR DESCRIPTION
**Pull request by alex-wilmer**

### CODE REVIEW

File: app/react-components/Mutation.js [hunk 1]
```diff
@@ -1,15 +1,17 @@
+/* global $ */
```
Comment (Jephuff) [inline app/react-components/Mutation.js]: Should probably import jQuery instead of using the global?

File: app/react-components/Mutation.js [hunk 2]
```diff
@@ -57,320 +61,322 @@ let Mutation = (() => {
...
-      );
-    }
-  }
-})()
+        </Column>
+        <Column style={{ ...styles.card, marginTop: '2rem' }}>
+          <ProteinLolliplotComponent
+            gene={gene}
+            $scope={$scope}
+            reset={this.state.ProteinLolliplot.reset}
```
Comment (Jephuff) [inline app/react-components/Mutation.js]: I think this is suppose to be `props.ProteinLolliplot.reset`

Comment (alex-wilmer) [inline app/react-components/Mutation.js]: oh nice good call

File: app/react-components/Mutation.js (hunk 1 above)
Comment (Jephuff) [inline app/react-components/Mutation.js]: We do have it https://github.com/NCI-GDC/portal-ui/blob/next/package.json#L118
I'm for removing it entirely

**Comment by alex-wilmer** [comment]
@Jephuff r?

File: app/react-components/Mutation.js [hunk 3]
```diff
@@ -57,320 +60,322 @@ let Mutation = (() => {
...
+            height: 450,
+            domainWidth: this.props.$scope.geneTranscript.length_amino_acid,
+            mutationId: this.props.mutation.ssm_id,
+          }),
+        });
+      },
+    }),
+    lifecycle({
+      componentDidMount() {
+        this.props.renderProteinLolliplot();
```
Comment (Jephuff) [inline app/react-components/Mutation.js]: missed this, `renderProteinLolliplot` needs props

**Comment by Jephuff** [comment]
👍 3

**Pull request merged**